brigand/algorithms/remove.hpp

```diff
@@ -6,51 +6,53 @@
 =================================================================================================**/
 #pragma once
 
-#include <brigand/sequences/clear.hpp>
 #include <brigand/functions/lambda/apply.hpp>
+#include <brigand/algorithms/wrap.hpp>
+#include <brigand/algorithms/transform.hpp>
+#include <brigand/sequences/append.hpp>
+#include <brigand/sequences/clear.hpp>
 #include <type_traits>
 
 namespace brigand
 {
-  namespace detail
-  {
-    template< class L1, typename Pred, class L2 = clear<L1>>
+namespace detail
```

Comment:
could be cool if those spacing change didn't get in as they make a lot of noise and derail PR review.